components/rmrk/Gallery/Gallery.vue

```diff
@@ -422,6 +421,10 @@ export default class Gallery extends mixins(PrefixMixin) {
         }
       }
 
+      .gallery__image-wrapper img {
```

Comment:
To the future Viki (won't it break the other views?)

Reply:
why would it?